x/concentrated-liquidity/incentives.go

```diff
@@ -552,6 +553,7 @@ func (k Keeper) GetAllIncentiveRecordsForUptime(ctx sdk.Context, poolId uint64,
 }
 
 // GetUptimeGrowthInsideRange returns the uptime growth within the given tick range for all supported uptimes
+// WARNING: this method may mutate the pool, make sure to refetch the pool after calling this method.
```

Comment:
Same question on this one, is not clear to me which part of this is mutative.

Reply:
added comment